osu.Game/Rulesets/Mods/IMod.cs

```diff
@@ -39,6 +39,18 @@ public interface IMod : IEquatable<IMod>
         /// </summary>
         bool UserPlayable { get; }
 
+        /// <summary>
+        /// Whether this mod is playable in a multiplayer match.
+        /// Should be <c>false</c> for mods that affect the gameplay progress based on user input (e.g. <see cref="ModAdaptiveSpeed"/>).
+        /// </summary>
+        bool PlayableInMultiplayer { get; }
+
+        /// <summary>
+        /// Whether this mod is valid to be a "free mod" in a multiplayer match.
+        /// Should be <c>false</c> for mods that affect the gameplay progress (e.g. <see cref="ModRateAdjust"/> and <see cref="ModTimeRamp"/>).
+        /// </summary>
+        bool ValidFreeModInMultiplayer { get; }
```

Comment:
Was hoping for a better and shorter name than this one (like `PlayableInMultiplayer`), but couldn't come up with anything better than this.